python/lsst/afw/geom/skyWcs/skyWcsContinued.py

```diff
@@ -20,9 +20,80 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
-__all__ = []
+import numpy as np
 
+from lsst.utils import continueClass
 from ..python import reduceTransform
 from .skyWcs import SkyWcs
 
+__all__ = []
+
+
+@continueClass  # noqa: F811
+class SkyWcs:
+    def pixelToSkyArray(self, x, y, degrees=False):
+        """
+        Convert numpy array pixels (x/y) to numpy array sky (ra/dec)
+        positions.
+
+        Parameters
+        ----------
+        x : `np.ndarray`
+            Array of x points.
+        y : `np.ndarray`
+            Array of y points.
+        degrees : `bool`, optional
+            Return ra/dec arrays in degrees if True.
+
+        Returns
+        -------
+        ra : `np.ndarray`
+            Array of Right Ascension.  Units are radians unless
+            degrees=True.
+        dec : `np.ndarray`
+            Array of Declination.  Units are radians unless
+            degrees=True.
+        """
+        xy = np.vstack((x, y))
+        rd = self.getTransform().getMapping().applyForward(xy)
+        rd[0, :] = rd[0, :] % (2.*np.pi)
+
+        if degrees:
+            return np.rad2deg(rd[0, :]), np.rad2deg(rd[1, :])
+        else:
+            return rd[0, :], rd[1, :]
+
+    def skyToPixelArray(self, ra, dec, degrees=False):
+        """
+        Convert numpy array sky (ra/dec) positions to numpy array
+        pixels (x/y).
+
+        Parameters
+        ----------
+        ra : `np.ndarray`
+            Array of Right Ascension.  Units are radians unless
+            degrees=True.
+        dec : `np.ndarray`
+            Array of Declination.  Units are radians unless
+            degrees=True.
+        degrees : `bool`, optional
+            Input ra/dec arrays are degrees if True.
+
+        Returns
+        -------
+        x : `np.ndarray`
+            Array of x points.
+        y : `np.ndarray`
+            Array of y points.
+        """
+        if degrees:
+            rd = np.deg2rad(np.vstack((ra, dec)))
```

Comment:
I think it would make sense to first denote the output from `vstack` as `rd`, and then convert it to degrees if `degrees`=True, rather than having an `else` statement.